### ISSUE TITLE
Recognize that get_defined_vars() takes unused parameters

### ISSUE THREAD

**Issue by sybrew** [WordPress/WordPress-Coding-Standards#1745]
## Is your feature request related to a problem?

Code speaks a thousand words.

```php
function some_function( $var_1 ) { // VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable

    $var_2 = get_something(); // VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable

    do_something( get_defined_vars() ); // this should negate above warnings.
}
```

## Describe the solution you'd like
Let `get_defined_vars()` be recognized as something that uses all, otherwise unused, vars.

**Comment by sybrew** [comment]
Forwarded issue to the responsible repo: sirbrillig/phpcs-variable-analysis/issues/91
